core/modules/filter/filter.pages.inc

```diff
@@ -536,10 +536,16 @@ function filter_format_editor_dialog_save($form, &$form_state) {
  */
 function _filter_format_editor_link_url_validate(&$element, &$form_state) {
   $value = trim($element['#value']);
+  
+  // Use functions in the link module to produce an external URL.
+  if(module_exists(link)){
```

Comment:
Need space after `if` and after closing bracket.

Reply:
I would have assumed that we'd want to enforce link module enabled in order to embed a link at all.

Reply:
@herbdool thanks, I will add the spaces.
I'm not sure about enforcing link module because we haven't done so hitherto.

Reply:
`'link'` should also be in (single) quotes.
The Link module just provides extra validation and fields and stuff related to hyperlinks but it isn't required for having links, so users can disable it. So that conditional should be there.

Reply:
@docwilmot thanks.
These comments actually apply to issue 3083 not 3085. Apologies for the confusion I caused by initially committing the changes in the wrong PR.